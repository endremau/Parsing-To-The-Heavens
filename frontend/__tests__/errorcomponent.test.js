import React from 'react';
import renderer from 'react-test-renderer'
import Error from '../src/components/errorcomponent'

it('renders correctly', ()=>{
    const tree=renderer
        .create(<Error/>)
        .toJSON();
    expect(tree).toMatchSnapshot()
})