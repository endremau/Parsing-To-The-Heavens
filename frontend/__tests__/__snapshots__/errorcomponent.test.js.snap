// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  style={
    Object {
      "width": "100%",
    }
  }
>
  <img
    alt="Logo"
    src="test-file-stub"
    style={
      Object {
        "width": "100%",
      }
    }
  />
</div>
`;
